whois-nrtm4-client/src/test/resources/mock/nrtm-snapshot.1.RIPE.json

```diff
@@ -0,0 +1,5 @@
+{"nrtm_version":4,"type":"snapshot","source":"RIPE","session_id":"4521174b-548f-4e51-98fc-dfd720011a0c","version":1}
```

Comment:
Are there NULL bytes in this file?

Reply:
no, there are no nulls.
This is a copy paste of the uncompressed snapshot (taking just a few records)